symplyphysics/laws/conservation/mixture_mass_equal_sum_of_components_masses.py

```diff
@@ -0,0 +1,32 @@
+from typing import Sequence
+from sympy import (Eq, solve)
+from symplyphysics import (units, Quantity, print_expression, Symbol, validate_input,
+    validate_output)
+from symplyphysics.core.operations.sum_array import SumArray
+from symplyphysics.core.symbols.symbols import tuple_of_symbols
+
+# Description
+## The mass of a mixture of liquids (gases) is equal to the sum of the masses of the components of the mixture
```

Comment:
As soon as this law is ready, you can refactor examples, that explicitly sum masses, eg melting ice example.

Reply:
Yes, i will do it.